android/src/main/java/ee/forgr/capacitor_updater/CapacitorUpdater.java

```diff
@@ -43,29 +43,32 @@ interface Callback {
 
 public class CapacitorUpdater {
     private static final String AB = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
-    private static SecureRandom rnd = new SecureRandom();
+    private static final SecureRandom rnd = new SecureRandom();
 
     private static final String DOWNLOADED_SUFFIX = "_downloaded";
     private static final String NAME_SUFFIX = "_name";
     private static final String STATUS_SUFFIX = "_status";
 
     private static final String FALLBACK_VERSION = "pastVersion";
     private static final String NEXT_VERSION = "nextVersion";
-    private final String bundleDirectory = "versions";
+    private static final String bundleDirectory = "versions";
+
+    public static final String TAG = "Capacitor-updater";
+    public static final String pluginVersion = "4.0.0";
```

Comment:
I think we should make this `PLUGIN_VERSION` at some point now that it is static.